src/main/java/com/salesforce/mirus/config/SourceConfig.java

```diff
@@ -43,8 +44,23 @@ public String getTopicsRegex() {
     return simpleConfig.originalsWithPrefix("consumer.");
   }
 
-  public String getDestinationBootstrapServers() {
-    return simpleConfig.getString(SourceConfigDefinition.DESTINATION_BOOTSTRAP_SERVERS.key);
+  private Map<String, Object> getDestinationProperties() {
+    // handle destination.bootstrap.server separately
+    String destBootstrap =
+        simpleConfig.getString(SourceConfigDefinition.DESTINATION_BOOTSTRAP_SERVERS.key);
+    Map<String, Object> destConsumerProps =
+        simpleConfig.originalsWithPrefix("destination.consumer.");
+    destConsumerProps.computeIfAbsent(
+        "bootstrap.servers", s -> destConsumerProps.put("bootstrap.servers", destBootstrap));
+    return destConsumerProps;
+  }
+
+  public Map<String, Object> getDestinationConsumerConfigs() {
```

Comment:
I think merging the source and destination consumer properties is really a "destination consumer" implementation detail, so I think this logic should live in `newDestinationConsumer()`. That way SourceConfig doesn't know anything about destination consumer.